main.nf

```diff
@@ -251,6 +263,62 @@ process get_software_versions {
     """
 }
 
+/*
+ * Write out models supported by Fred2 (for installed predictor tool versions)
```

Comment:
Maybe it would make more sense to have this at the beginning of `main.nf` at just exit after performing this? Then we also don't have to care if users specify additional parameters/inputs?